### ISSUE TITLE
Bullet & Numbering

### ISSUE THREAD

**Issue by qimuxi1990** [iarna/rtf-parser#15]
Dear @iarna 
I just discover that fcharset2 Symbol is not supported here.
Due to the fact that it does not related to any [window codepage](https://msdn.microsoft.com/en-us/library/cc194829.aspx). So it will never be handled by iconv.
However in rtf documents, bullet and numbering are frequently. I believe it would be pretty nice to have support on Symbols or simply Bullet & Numbering.
Here is an example rtf string:

> {\rtf1\fbidis\ansi\ansicpg1252\deff0\deflang1033{\fonttbl{\f0\fnil\fcharset0 Microsoft Sans Serif;}{\f1\fswiss\fprq2\fcharset0 Tahoma;}{\f2\fnil\fcharset2 Symbol;}}  {\colortbl ;\red0\green0\blue0;}  \viewkind4\uc1\pard{\pntext\f2\'B7\tab}{\*\pn\pnlvlblt\pnf2\pnindent0{\pntxtb\'B7}}\ltrpar\cf1\f0\fs16 Some Text \cf0\f1\fs20 Some text.\cf1\f0\fs16\par  \cf0\f1\fs20{\pntext\f2\'B7\tab}some text.\cf1\f0\fs16\par  }  

Same thing possibly also happens to fcharset1 DEFAULT, and fcharset255 OEM. But I have not seen examples from my experience.
Thank you.

**Comment by qimuxi1990** [comment]
Also emoji. I see one emoji is expressed with `\u55357?\u56842?` which will result in a break in interpreter `ctrl$u` function.

**Comment by iarna** [comment]
@qimuxi1990 Yeah, I agree! I'd love to see these fixed (my use case has been very limited, thus the overall limited implementation here).

Would you be interested in being  co-maintainer of this project? If yes, I'll add you here and on npm! (But you'll have to tell me your npm account name. 😀) Please feel free to email me at `me@re-becca.org` if you'd like to discuss this before making a decision.

**Comment by qimuxi1990** [comment]
@iarna Thank you very much for inviting me but I think I would not have sufficient time to be a co-maintainer and I did not got an npm account yet. ;-)
However, since the company I am working for is currently building a CRM with RTF support for in-house purpose. I may see more use cases here.
If I witness some common cases or figure out any improvement. I'll also update them here.

**Comment by iarna** [comment]
Yeah, I get that. Mostly I just wanted to give you the ability to land your patches, when you have them, on your own, as my own ability to be here is limited too.

**Comment by markb-trustifi** [comment]
It can be fixed by changing this function:
```
flushHexStore () {
    if (this.hexStore.length > 0) {
      let charset = this.group.get('charset');
      let hexstr = this.hexStore.map(cmd => cmd.value).join('')
      let translate = charset === 'SYMBOL' ? '&#x'+hexstr+';' : iconv.decode(Buffer.from(hexstr, 'hex'), charset)
      this.group.addContent(new RTFSpan({value: translate}))
      this.hexStore.splice(0)
    }
  }
```
